### PR TITLE
Krypton Keyboard layout skins fix 13.col visible

### PR DESCRIPTION
**Pull request by Wolftronics-SBC**
Hungarian.xml first row 13 characters
## Types of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Improvement (non-breaking change which improves existing functionality)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the [Code guidelines](https://codedocs.xyz/xbmc/xbmc/code_guidelines.html) of this project 
- [ ] My change requires a change to the documentation, either Doxygen or wiki
- [ ] I have updated the documentation accordingly
- [ ] I have read the [CONTRIBUTING](https://github.com/xbmc/xbmc/blob/master/CONTRIBUTING.md) document
- [ ] I have added tests to cover my change
- [ ] All new and existing tests passed

### CODE REVIEW

File: addons/skin.estouchy/xml/DialogKeyboard.xml [hunk 1]
```diff
@@ -169,6 +169,18 @@
 						<ondown>131</ondown>
 						<include>KeyboardButton</include>
 					</control>
+					<control type="button" id="112">
+						<description>(0,12) key button</description>
+						<onup>306</onup>
+						<ondown>132</ondown>
+						<include>KeyboardButton</include>
+					</control>
+					<control type="button" id="112">
```
Comment (phil65) [inline addons/skin.estouchy/xml/DialogKeyboard.xml]: two buttons with same id?

File: addons/skin.estuary/xml/DialogKeyboard.xml [hunk 2]
```diff
@@ -157,6 +157,20 @@
...
 						<include>KeyboardButton</include>
 					</control>
+					<control type="button" id="112">
+						<description>(0,12) key button</description>
+						<onup condition="!Integer.IsGreater(Container(9010).NumItems,0)">8</onup>
+						<onup condition="Integer.IsGreater(Container(9010).NumItems,0)">9010</onup>
+						<ondown>132</ondown>
+						<include>KeyboardButton</include>
+					</control>
+					<control type="button" id="112">
```
Comment (phil65) [inline addons/skin.estuary/xml/DialogKeyboard.xml]: same here.

Comment (Wolftronics-SBC) [reply addons/skin.estuary/xml/DialogKeyboard.xml]: Krypton:Updated hungarian.xml frequently used buttons add to first keyboard layout... 1eb3981 no two buttons....

**Comment by MartijnKaijser** [comment]
We have released the last v17 version so I will close this. Our master branch for v18 is still open for this to go in. Thx

**Comment by Wolftronics-SBC** [comment]
v17 close or not close?  @MartijnKaijser

**Pull request closed without merging**